src/ImageSharp/MetaData/Profiles/ICC/DataReader/IccDataReader.cs

```diff
@@ -11,7 +11,7 @@ namespace SixLabors.ImageSharp.MetaData.Profiles.Icc
     /// </summary>
     internal sealed partial class IccDataReader
     {
-        private static readonly Encoding AsciiEncoding = Encoding.GetEncoding("ASCII");
+        private static readonly Encoding AsciiEncoding = Encoding.ASCII;
```

Comment:
Wondering if we really need this as a property instead of just using `Encoding.ASCII` where we need it. Or maybe rename the field?

Reply:
Updated!